src/PhpSpec/CodeAnalysis/AccessInspectorInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Phpspec\CodeAnalysis;
+
+interface AccessInspectorInterface
```

Comment:
Wait I see what you're saying.

We could have `MagicAccessInspector` decorate `VisibilityAccessInspector`. That way MageSpec just needs to mess with the container config and remove that decorator.